packages/tpetra/core/compat/Tpetra_KokkosCompat_ClassicNodeAPI_Wrapper.hpp

```diff
@@ -69,16 +69,26 @@ class KokkosDeviceWrapperNode {
 };
 
 #ifdef KOKKOS_ENABLE_SYCL
+#ifdef HAVE_TPETRA_SHARED_ALLOCS
+  typedef KokkosDeviceWrapperNode<::Kokkos::Experimental::SYCL, ::Kokkos::Experimental::SYCLSharedUSMSpace> KokkosSYCLWrapperNode;
+#else
   typedef KokkosDeviceWrapperNode<::Kokkos::Experimental::SYCL, ::Kokkos::Experimental::SYCLDeviceUSMSpace> KokkosSYCLWrapperNode;
 #endif
+#endif
 
 #ifdef KOKKOS_ENABLE_HIP
+  // NOTE: KokkosKernels does not currently suport HIPManagedSpace, but
+  // if it did, we'd add HAVE_TPETRA_MANAGED_ALLOCS here
```

Comment:
would it be better to throw a cmake configure error, e.g., "HIP shared space not supported yet" if the user enables HIP and also requests the default allocation to be in shared space?

Reply:
ETI for ``<HIP, HIPManagedSpace>`` doesn't exist, but everything should still work with it, AFAIK. It just might take longer to build downstream packages. We will add the ETI option.

Reply:
So in other words, this option should be enabled for `HIPManagedSpace` as well?